tests/test_wps_climdexInput_csv.py

```diff
@@ -1,14 +1,19 @@
 import pytest
 from tempfile import NamedTemporaryFile
+from pkg_resources import resource_filename
+from wps_tools.file_handling import csv_handler
 
 from wps_tools.testing import run_wps_process, local_path, process_err_test
 from quail.processes.wps_climdexInput_csv import ClimdexInputCSV
 
+tmax_file_content = csv_handler(resource_filename("tests", "data/1018935_MAX_TEMP.csv"))
+tmin_file_content = csv_handler(resource_filename("tests", "data/1018935_MIN_TEMP.csv"))
+prec_file_content = csv_handler(
+    resource_filename("tests", "data/1018935_ONE_DAY_PRECIPITATION.csv")
+)
```

Comment:
It make work better to make these `conftest.py` fixtures rather than global variables.